redis-om-spring/src/main/java/com/redis/om/spring/search/stream/AggregationStreamImpl.java

```diff
@@ -395,14 +395,14 @@ public AggregationStream<T> cursor(int count, Duration timeout) {
   public <R extends T> Slice<R> toList(PageRequest pageRequest, Class<?>... contentTypes) {
     applyCurrentGroupBy();
     aggregation.cursor(pageRequest.getPageSize(), -1);
-    return (Slice<R>) new AggregationPage<>(this, pageRequest, entityClass, gson, mappingConverter, isDocument );
+    return new AggregationPage(this, pageRequest, entityClass, gson, mappingConverter, isDocument );
```

Comment:
I knew it's better. Is it changed in latest Java?
```suggestion
    return new AggregationPage<>(this, pageRequest, entityClass, gson, mappingConverter, isDocument);
```

Reply:
I don't quite know why byt removing the cast to `(Slice<R>)` means that adding `<>` on the `AggregationPage` new results in:
```
[ERROR] /Users/bsb/Code/module-clients/java/high-level/redis-om/redis-om-spring/redis-om-spring/src/main/java/com/redis/om/spring/search/stream/AggregationStreamImpl.java:[398,12] cannot infer type arguments for com.redis.om.spring.search.stream.AggregationPage<>
[ERROR] /Users/bsb/Code/module-clients/java/high-level/redis-om/redis-om-spring/redis-om-spring/src/main/java/com/redis/om/spring/search/stream/AggregationStreamImpl.java:[405,12] cannot infer type arguments for com.redis.om.spring.search.stream.AggregationPage<>
```

Java Generics, just when I think I understand them I realize I don't!